<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="zhouruideMacBook-Pro-2.local" ignored="0" name="注册接口" tests="6" failures="6" timestamp="31 一月 2021 15:18:27 GMT" time="4.898" errors="0">
  <testcase name="test" time="0.803" classname="cases81.RegisterCase2">
    <failure type="java.lang.AssertionError" message="expected [PASSED] but found [FAILED]">
      <![CDATA[java.lang.AssertionError: expected [PASSED] but found [FAILED]
	at cases81.RegisterCase2.test(RegisterCase2.java:44)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.113" classname="cases81.RegisterCase2">
    <failure type="java.lang.AssertionError" message="expected [PASSED] but found [FAILED]">
      <![CDATA[java.lang.AssertionError: expected [PASSED] but found [FAILED]
	at cases81.RegisterCase2.test(RegisterCase2.java:44)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.105" classname="cases81.RegisterCase2">
    <failure type="java.lang.AssertionError" message="expected [PASSED] but found [FAILED]">
      <![CDATA[java.lang.AssertionError: expected [PASSED] but found [FAILED]
	at cases81.RegisterCase2.test(RegisterCase2.java:44)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.091" classname="cases81.RegisterCase2">
    <failure type="java.lang.AssertionError" message="expected [PASSED] but found [FAILED]">
      <![CDATA[java.lang.AssertionError: expected [PASSED] but found [FAILED]
	at cases81.RegisterCase2.test(RegisterCase2.java:44)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="1.253" classname="cases81.RegisterCase2">
    <failure type="java.lang.AssertionError" message="expected [PASSED] but found [FAILED]">
      <![CDATA[java.lang.AssertionError: expected [PASSED] but found [FAILED]
	at cases81.RegisterCase2.test(RegisterCase2.java:44)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
  <testcase name="test" time="0.888" classname="cases81.RegisterCase2">
    <failure type="java.lang.AssertionError" message="expected [PASSED] but found [FAILED]">
      <![CDATA[java.lang.AssertionError: expected [PASSED] but found [FAILED]
	at cases81.RegisterCase2.test(RegisterCase2.java:44)
... Removed 35 stack frames]]>
    </failure>
  </testcase> <!-- test -->
</testsuite> <!-- 注册接口 -->
